Makefile

```diff
@@ -63,7 +70,7 @@ ci: builddeps buildkitd
 builddeps: gitmeta buildctl
 
 gitmeta:
```

Comment:
should it be `$(BINDIR)/gitmeta` ? so that make usual dependencies rules work?

Reply:
👍 it should